### PR TITLE
Skip the remote executor tests

### PR DESCRIPTION
**Pull request by alexperovich**
https://github.com/dotnet/arcade/issues/7092

### CODE REVIEW

**Review by ChadNedzlek — APPROVED** [review]

File: src/Microsoft.DotNet.RemoteExecutor/tests/RemoteExecutorTests.cs [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
 using Xunit;
-using Xunit.Abstractions;
 using Xunit.Sdk;
 
 namespace Microsoft.DotNet.RemoteExecutor.Tests
 {
     public class RemoteExecutorTests
     {
-        [Fact]
+        [Fact(Skip = "Remote executor is broken in VS test explorer")]
```
Comment (MattGal) [inline src/Microsoft.DotNet.RemoteExecutor/tests/RemoteExecutorTests.cs:13]: How does this not disable it from normal console execution?

Comment (ChadNedzlek) [reply src/Microsoft.DotNet.RemoteExecutor/tests/RemoteExecutorTests.cs:13]: Is that even possible?

Comment (alexperovich) [reply src/Microsoft.DotNet.RemoteExecutor/tests/RemoteExecutorTests.cs:13]: its only possible by basically rewriting swaths of xunit to make a custom Fact attribute.

Comment (MattGal) [reply src/Microsoft.DotNet.RemoteExecutor/tests/RemoteExecutorTests.cs:13]: I thought runtime did this all the time using some kind of extension pattern.  Just seems like perma-skip = never run and I understand they pass outside of VS.

Comment (ChadNedzlek) [reply src/Microsoft.DotNet.RemoteExecutor/tests/RemoteExecutorTests.cs:13]: Runtime has a huge swath of custom fact stuff that I don't think we want to duplicate just to deal with these handful of tests. :-)

**Review by jakubstilec — APPROVED** [review]

**Pull request merged**